index.bs

```diff
@@ -378,37 +340,63 @@ When requesting the `webid` scope, the user's WebID MUST be present in the ID To
             "webid": "https://janedoe.com/web#id",
             "iss": "https://idp.example.com",
             "sub": "janedoe",
-            "aud": "https://client.example.com/web#id",
-            "nonce": "n-0S6_WzA2Mj",
-            "exp": 1311281970,
+            "aud": ["https://client.example.com/web#id", "solid"],
+            "azp": "https://client.example.com/web#id",
             "iat": 1311280970,
+            "exp": 1311281970,
+            "cnf":{
+              "jkt":"0ZcOCORZNYy-DWpqq30jZyJGHTN0d2HglBV3uiguA4I"
+            }
         }
     </pre>
 </div>
 
 # Resource Access # {#resource}
 
+## Authorization Server Discovery ## {#authorization-server-discovery}
+
+When client performs unauthenticated request to protected resource,
+Resource Server MUST respond with HTTP <code>401</code> status code,
+as well as <code>WWW-Authenticate</code> HTTP header including <code>UMA</code>
```

Comment:
Should the reference to UMA be part of the normative text ? I would have expected something along the lines of "as well as <code>WWW-Authenticate</code> HTTP header including an authentication scheme allowing the discovery of the authorization server.", and then to have a non-normative example instantiating this generic approach with UMA. This way, Solid-OIDC remains authentication-focused, and orthogonal to the chosen authorization method.

Reply:
We definitely want any mention of UMA to be in non-normative text. It should just be one example among (potentially) many possibilities.

Reply:
If we don't specify (have it non-normative) how exactly RS advertises its AS, we can't expect interoperable implementations. We could explore an approach similar to https://github.com/solid/notifications where we would have alternative ways of doing specific parts of the protocol. Still, each alternative should have a normative text, including how support can be discovered.